packages/forms/src/lib/datepicker/README.md

```diff
@@ -0,0 +1,95 @@
+# @acpaas-ui/ngx-components/forms
+The package creates a custom input component allowing the user to select a date either by input or by picking one in the calendar flyout.
+
+## Dependencies
```

Comment:
I'm not sure if we should list these as dependencies (with the exception of `@acpaas-ui/js-date-utils`).

Reply:
It's in line with the other examples, but we can always review it.

Reply:
No, it's only in line with the one's we overlooked, i.e.: `table` and `flyout button`. The packages are in the same repository, so shouldn't be listed as dependency.